hackathon_site/hardware/views.py

```diff
@@ -101,7 +109,10 @@ def get_permissions(self):
         if self.request.method == "POST":
             return [UserHasProfile()]
         if self.request.method == "GET":
-            return [FullDjangoModelPermissions()]
+            if self.request.user.has_perm("hardware.change_order"):
```

Comment:
Can we just Or these two permissions together? `return [FullDjangoModelPermissions() | UserHasProfile()]` might work